### DISCUSSION TITLE
v1.4.3

### DISCUSSION THREAD

**Discussion by MrMeowzz** [MrMeowzz/Funkin-MrMeowzz#31] [Updates]
- added warning, stun, and poison must hit sustain notes
- fixed note type dropdown interfering with fps counter when enabled
- made fire notes and halo notes harder to hit
- ugh hard plus has different chart with note types [charted by [DeceptedParasite](https://github.com/DeceptedParasite)]

<hr /><em>This discussion was created from the release <a href='https://github.com/MrMeowzz/Funkin-MrMeowzz/releases/tag/v1.4.3'>v1.4.3</a>.</em>

**Comment by DeceptedPar** [comment]
thanks for adding it in man!
